doc/release-notes.md.in

```diff
@@ -52,8 +50,29 @@ AtomVM currently supports the following versions of ESP-IDF:
 | ESP-IDF v4.2 | ✅ |
 | ESP-IDF v4.3 | ✅ |
 | ESP-IDF v4.4 | ✅ |
-| ESP-IDF v5.0 | ❌ |
+| ESP-IDF v5.0 | ✅ |
 
```

Comment:
Should we also add 5.1?

Reply:
5.1 is still and rc

Reply:
You are right, I got a little ahead of myself. It won't be released until this coming week. ;)

Reply:
Actually it was released on Friday. June 30th.

Reply:
Let's update the release notes when we build off the 5.1 release.